scripts/installer/Dockerfile

```
@@ -4,4 +4,9 @@ RUN yum -y install epel-release https://yum.puppetlabs.com/puppet5/puppet5-relea
 
 ARG FOREMAN_VERSION
 RUN yum -y install https://yum.theforeman.org/releases/$FOREMAN_VERSION/el7/x86_64/foreman-release.rpm
+RUN yum -y install foreman-release-scl
```

This package was dropped in the latest version, but for the past few releases it has only depended on `centos-release-scl-rh`
```suggestion
RUN yum -y install centos-release-scl-rh
```